ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -254,114 +256,115 @@ private[testtool] abstract class CommandDeduplicationBase(
 
   // staticTime - we run calls in parallel and with static time we would need to advance the time,
   //              therefore this cannot be run in static time
-  if (!staticTime)
-    testGivenAllParticipants(
```

Comment:
this test only makes use of the enabled/disabled test feature, no actual changes in the test itself